### DISCUSSION TITLE
artisan activitylog:clean + optimize table table_name

### DISCUSSION THREAD

**Discussion by irvine48** [spatie/laravel-activitylog#870] [Ideas]
Hi devs,

I use this activity log a lot, i found that my logs getting extreamly big, i try to doing a clean up first time since two year and found that the table size not being reduce even the clean is being execute.

maybe document need mention about optimize table need to be call in mysql to clear up the space.

**Comment by Gummibeer** [comment]
Feel free to add this to the docs in a PR. I have no idea if that's really required or if MySQL would do it on its own after some time - but I think that it can't hurt to add it as a notice.
